### PR TITLE
Cig fix, now every 10 seconds

### PR DESCRIPTION
**Pull request by Jan-Blasiak**
## About The Pull Request
It's a fix for the cigarette bug that pushed all ingredients in 25 seconds. Now it's one tick every 10 seconds

## Screenshots
smoketime

### CODE REVIEW

**Comment by Jan-Blasiak** [comment]
Dangit, it definitely shouldn't, it's 300 seconds, tick every 10 seconds, so it should be 5 more to spare from 0,2 per tick!

**Comment by Jan-Blasiak** [comment]
Unless first tick is somehow omitted, did you light it up in your mouth Torino?

**Comment by TorinoFermic** [comment]
Yes with a flare.

**Comment by Jan-Blasiak** [comment]
Hmmm...

I'll check it tomorrow, it's 2:22 here.

**Comment by Jan-Blasiak** [comment]
I mean... Technically it starts from 299, because of decrement at the beginning, but it still should be giving 25...

Consider me currently baffled.

I may simply add a second to every cig burn to 

""""""""""Fix"""""""""" 

The problem.

**Comment by Jan-Blasiak** [comment]
Yup, adding 1 second fixes it.

(WHYYYYYYYYYYYYY)

**Comment by Jan-Blasiak** [comment]
cev — kopia — kopia (2)

Fixing the amount of ingredients

**Comment by Jan-Blasiak** [comment]
All cigs fixed now, reduced duration of cigars, now all have proper amount of ingredients.

**Comment by Jan-Blasiak** [comment]
The reason I've reduced duration, is that to make Havana cigs last the 7000 seconds, those would have to contain more ingredients than a large beaker (140u), which is absolutely ridiculous.

**Comment by TorinoFermic** [comment]
It wasn't needed to add the volume by 1 or the smoke time to be XX1. All reagents are already gone, that's because the tasting message fails the minimum threshold which happens to be 0.2

**Comment by Jan-Blasiak** [comment]
Ah, ok, thanks...

File: code/game/objects/items/weapons/cigs_lighters.dm [hunk 1]
```diff
@@ -281,8 +283,9 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	type_butt = /obj/item/trash/cigbutt/cigarbutt
 	throw_speed = 0.5
 	item_state = "cigaroff"
-	smoketime = 1500
+	smoketime = 701
```
Comment (TorinoFermic) [inline code/game/objects/items/weapons/cigs_lighters.dm]: 700

File: code/game/objects/items/weapons/cigs_lighters.dm [hunk 2]
```diff
@@ -303,8 +306,9 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	icon_state = "cigar2off"
 	icon_on = "cigar2on"
 	icon_off = "cigar2off"
-	smoketime = 7200
+	smoketime = 1001
```
Comment (TorinoFermic) [inline code/game/objects/items/weapons/cigs_lighters.dm]: 1000

File: code/game/objects/items/weapons/cigs_lighters.dm [hunk 3]
```diff
@@ -339,7 +343,8 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	icon_on = "pipeon"  //Note - these are in masks.dmi
 	icon_off = "pipeoff"
 	smoketime = 0
-	chem_volume = 50
+	chem_volume = 51
```
Comment (TorinoFermic) [inline code/game/objects/items/weapons/cigs_lighters.dm]: 50

Comment (kperdue2002) [reply code/game/objects/items/weapons/cigs_lighters.dm]: it'd be a good idea to comment why the values are 1 above what they should be

**Comment by Jan-Blasiak** [comment]
I've had 3h of sleep.
Another day

File: code/game/objects/items/weapons/cigs_lighters.dm (hunk 3 above)
Comment (TorinoFermic) [reply code/game/objects/items/weapons/cigs_lighters.dm]: To explain why, the author was concerned about the number of tasting nicotine messages being one off and added one to all them to fix it until I tell him that's not the solution and the code that emits these messages has thresholds which the last message will not go out due it. You can check it at code\modules\mob\living\carbon\taste.dm

**Comment by TorinoFermic** [comment]
Okay, I know why the last message out of 25 wasn't out.

`	if(smoketime < 1)
		die()
		return` causes it to die early, you can change it to 0

**Comment by MadmanMartian** [comment]
Could move the smoketime-- to after the inhalation checks

**Comment by Jan-Blasiak** [comment]
Ok, I've changed time to smoke and burnout condition, turning it to <0 worked.

cig final

**Pull request merged**